### PR TITLE
Add version info to summary, etc

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: dart_test.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+tags:
```
Comment (isoos) [inline dart_test.yaml]: Is this a config file that some Dart tooling reads?

Comment (kevmoo) [inline dart_test.yaml]: pkg/test

**Comment by kevmoo** [comment]
> My only question is the config file, otherwise lgtm

What's your question?

**Comment by isoos** [comment]
>> My only question is the config file, otherwise lgtm
> What's your question?

dart_test.yaml, but you have answered that.

**Pull request merged**